spec/integration/outputs/parent_spec.rb

```diff
@@ -43,29 +44,122 @@
         insist { cur_count } == event_count
       end
     end
-end
+  end
 
-describe "(http protocol) index events with static parent", :integration => true do
-  it_behaves_like 'a parent indexer' do
-    let(:parent) { "test" }
-    let(:config) {
-      {
-        "hosts" => get_host_port,
-        "index" => index,
-        "parent" => parent
+  describe "(http protocol) index events with static parent" do
```

Comment:
I just moved the integration declaration to the context that envelops this describe block. that should be enough, right?